graphql2/schema.graphql

```diff
@@ -97,6 +97,9 @@ type Query {
 
   # Returns a Slack channel with the given ID.
   slackChannel(id: ID!): SlackChannel
+
+  # Returns status of sendtest with given contact method id
+  sendTestStatus(cmID: ID!): NotificationState
```

Comment:
This implies it's sending a message. I'm thinking since contact methods are rate-limited by both test & verify messages -- should we mirror that here?

```suggestion
  testVerifyStatus(id: ID!): NotificationState
```

I'm unsure if we should use `cmID` or `id` (to match other APIs) we do indicate in the documentation/comment it's a contact method ID.

Reply:
On second thought maybe we can expand this a bit (pseudocode):

```graphql
lastTestVerify(id: ID!) {
  type: test | verify
  startedAt: ISOTimestamp
  state: NotificationState!
}
```

This would give use better information on what options are available to the user and the information we can provide.

When the dialog opens we could send a message if the last one was sent > 5m ago, otherwise we can display a "test message sent 3m ago" with a "Retry" button that's enabled after 1m -- getting rid of the "rate limit exceeded" error that provides no feedback.